spec/shared_examples/user_magic_login_shared_examples.rb

```diff
@@ -32,7 +32,8 @@
       end
 
       it do
-        TestMailerClass = Class.new # need a mailer class to test
+        test_mailer_class = Class.new # need a mailer class to test
+        stub_const('TestMailerClass', test_mailer_class)
         sorcery_model_property_set(:magic_login_mailer_class, TestMailerClass)
         expect(config.magic_login_mailer_class).to eq TestMailerClass
```

Comment:
It doesn’t seem necessary to define the constant here. For example, the test should still pass if we do it like this.

```ruby
test_mailer_class = Class.new
sorcery_model_property_set(:magic_login_mailer_class, test_mailer_class)
expect(config.magic_login_mailer_class).to eq test_mailer_class
```

It looks like we can apply the same fix to the remaining places that use stub_const.